content/legal/index.md

```diff
@@ -5,6 +5,6 @@ title: Legal
 constellation: constellation-01.svg
 ---
 
-[Report Copyright Infringement on the IPFS Gateway Service](https://docs.google.com/a/andyet.com/forms/d/1fXchtQEQ6WDjedSk46ZZiye8vl4UxvyRZiltSOv2huc/viewform)
+Report Copyright Infringement or Abuse on the IPFS Gateway Service: [abuse@ipfs.io](mailto:abuse@ipfs.io)
```

Comment:
What's the rational to change to email? Form gave the requests some structure.

Reply:
The form wasn't working, this is a temp workaround for now (it didn't provide an entry for providing URLs and didn't email notify us for new entries). I'm working with legal to put together a new structure for doing this, so expect this to change shortly.

Reply:
Understood. Will merge for now then.